oemoflex/helpers.py

```diff
@@ -23,13 +23,14 @@ def get_experiment_paths():
 
     """
     module_path = os.path.abspath(os.path.dirname(__file__))
-    path_config = os.path.join(module_path, 'experiment_paths.yml')
+    path_config = os.path.join(module_path, 'model_config', 'experiment_paths.yml')
 
     with open(path_config, 'r') as config_file:
         config = yaml.safe_load(config_file)
 
     # Use base path to make other paths absolute and drop it
     basepath = os.path.realpath(os.path.join(module_path, config.pop('base')))
+    config.pop('mapping')
     experiment_paths = {k: os.path.join(basepath, v) for k, v in config.items()}
```

Comment:
The need to drop fields is not so nice. I leave it as it his because I hope we can separate the project-related stuff from oemoflex-related soon. `base` and `mapping` will then vanish from here anyway.